test/integration/cli/help.bats

```diff
@@ -5,126 +5,126 @@ load ${BASE_TEST_DIR}/helpers.bash
 @test "cli: show info" {
   run machine
   [ "$status" -eq 0  ]
-  [[ ${lines[0]} =~ "Usage:"  ]]
-  [[ ${lines[1]} =~ "Create and manage machines running Docker"  ]]
+  [[ ${lines[0]} =~ "Usage:"  ]] || false
```

Comment:
I guess maybe `except the command following the final && or ||` gives us our out-of-jail-free card?